tests/reg_tests/test_solve.py

```diff
@@ -124,6 +124,9 @@ def test_solve(self):
 
 
 class TestFailedMesh(unittest.TestCase):
+
+    N_PROCS = 1
```

Comment:
Just FYI that this is already the default in testflo, but I guess this is more explicit.

Reply:
I think `N_PROCS = 1` will run with MPI but with one processor. The test in the terminal has `(mpi)` before the path and no other terminal output is printed. I made this change to avoid having the ADflow header printed out during the tests.